client/extensions/woocommerce/app/promotions/promotion-create.js

```diff
@@ -73,31 +86,55 @@ class PromotionCreate extends React.Component {
 	}
 
 	onSave = () => {
-		// TODO: Add action to save promotion.
+		const { site, promotion, translate } = this.props;
+
+		this.setState( () => ( { busy: true } ) );
+
+		const getSuccessNotice = () => {
+			return successNotice(
+				translate( '%(promotion)s promotion successfully created.', {
```

Comment:
`promotion.name` isn't defined for me so I just get this as a notice:

<img width="373" alt="screen shot 2017-10-31 at 11 05 44 am" src="https://user-images.githubusercontent.com/689165/32240901-d94d83c8-be2b-11e7-9082-31e218f31314.png">

This is the object I have at the time:

<img width="624" alt="screen shot 2017-10-31 at 11 08 50 am" src="https://user-images.githubusercontent.com/689165/32240925-eb734e8e-be2b-11e7-904a-6ec07cd56394.png">

It *does* work for updating.

Reply:
Addressed in #19370